web/components/admin/Navbar/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import { SecondaryButton } from '../../input/buttons';
+import logo from '../../../assets/logo.svg';
+
+class AdminNavbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tabIndex: 0,
+    };
+  }
+
+  changeTab = index => this.setState({ tabIndex: index });
+
+  render() {
+    const { pages } = this.props;
+    const { tabIndex } = this.state;
+    return (
+      <div>
+        <nav className="fill-width flex">
+          <div className="flex ai-center jc-start margin-sides-l">
+            <div className="flex ai-center">
+              <img alt="nwHacks" src={logo} />
```

Comment:
Idk how much we care about this, but the nwHacks logo for the internal tools is different in the Figma file (some blue variant). I feel like it better suits the general blue theme that our admin panel has.

Reply:
can we change it if someone brings it up LOL